pkg/escrow/resource.go

```diff
@@ -141,6 +142,9 @@ func (e Stellar) processReservationResources(resData workloads.ReservationData)
 	for _, k8s := range resData.Kubernetes {
 		rsuPerNodeMap[k8s.NodeId] = rsuPerNodeMap[k8s.NodeId].add(processKubernetes(k8s))
 	}
+	for _, vm := range resData.VirtualMachines {
```

Comment:
Yeah, as mentioned above, ReservationData is obsolete and if u try to find if this method is used it will only show up in test files. again the ReservationData structure was kept only for backward compatibility with some historic data on mainnet.
Yeah, explorer needs a lot of clean up but it's not done because of it's hopefully short life.

Reply:
While working on the explorer it's better if u trace code from api end point (not the one named legacy)